### DISCUSSION TITLE
Global variable acquisition problem

### DISCUSSION THREAD

**Discussion by minliuhua** [vuepress-theme-hope/vuepress-theme-hope#4626] [Q&A] [answered]
english ：I want to set global variables and retrieve them from markdown.
For example, how to obtain the version in packagejson in markdown, if it is obtained in a code block

Chinese：我想设置全局变量，然后markdown里面获取。
比如package.json中的version，如何在markdown中获取，如果在代码块中获取

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress/core/discussions/1622

Ask vuepress related question in its own repo
